README.md

```diff
@@ -0,0 +1,705 @@
+
+
+#### Step: Change date stamp in Containerfile (default and minimal only)
+
+> The step is skipped if image type is not 'default' or 'minimal'
+
+The step changes (*# Tags* with date stamp) corresponded [`Containerfiles/*/Containerfile.*`](https://github.com/AlmaLinux/container-images/tree/docker-library), which Docker will use to build images for the *Official Library*. An example is for AlmaLinux 8 minimal `Containerfiles/8/Containerfile.minimal` file:
+```docker
+# Tags: 8-minimal, 8.9-minimal, 8.9-minimal-20240319
+
+FROM quay.io/almalinuxorg/8-minimal:8.9-20240319
+```
+The change indicates that new `default` and/or `minimal` container image was pushed to *Client Library* and the same should be requested to build by Docker. The change will later be committed into `docker-file` branch.
+
+#### Step: Upload changed *Containerfiles/*/Containerfile.**
+
+> The step is skipped if image type is not 'default' or 'minimal'
+
+The step uses [actions/upload-artifact@v4](https://github.com/actions/upload-artifact) to store as artifact changed on previous step Containerfile. The artifact is named against `image_type`, like `containerfiles-image_types`.
+
+Artifacts are used to transfer files between different jobs of the same workflow. The artifact is zip archive of the file without file-path included.
+
+That is also possible to download artifacts via GitHub Action's web interface.
+
+### Job: Collect and save changed Containerfile(s) used by Docker Official Library
+
+> The job is skipped if image type is not 'default' or 'minimal'
+
+#### Step: Checkout *container-images*, branch 'docker-library'
+
+Checkouts *container-images* into branch 'docker-library'. The repository directory is located at `/home/runner/work/container-images/container-images`.
+The [actions/checkout@v4](https://github.com/actions/checkout/) is used.
+
+#### Step: Download changed Containerfiles
+
+Uses [actions/download-artifact@v4](https://github.com/actions/download-artifact) to download multiple (`merge-multiple: true`) artifacts with changed Containerfiles. The files are saved into the `Containerfiles/version_major/` directory.
+
+#### Step: Commit and push *Containerfiles/version_major/Containerfile.** changes
+
+> The step is skipped if '*Push to production registries*' is not checked (`inputs.production` set to `false`.)
+
+Uses [EndBug/add-and-commit@v9](https://github.com/marketplace/actions/add-commit) to commit and push Containerfiles, which was downloaded on the previous step, and changed by the previous job.
+
+The commit message is:
+```yaml
+AlmaLinux ${{ inputs.version_major }} image build as of ${{ needs.build.outputs.date_stamp }} (with ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}).
+```
+It includes AlmaLinux version major, image build date, and reference to this Github Action.
```

Comment:
It includes the AlmaLinux version major, image build date, and reference to this GitHub Action.

Reply:
Done.